shm/src/ship.cpp

```diff
@@ -0,0 +1,50 @@
+#include "ship.hpp"
+
+#include <iostream>
+#include <numeric>

```

Comment:
Here you have included headers,then standard library includes.

Reply:
Firstly we include header file that has declaration of class. Then standard libraries. Last but not least - additional headers that need to be used in that file.

Reply:
So I think that it is consistent